students/drovdahl/lesson02/grid.py

```diff
@@ -0,0 +1,76 @@
+#! /usr/local/bin/python3
+
+
+def grid():
+    '''
+    This function will draw a static grid with 4 quadrants that are 6x6.
+    A '+' character will be used in the corners
+    A '-' or '|' character will be used in between the corners
+    '''
+    print(2*('+' + 4*('-')) + '+')
+    print((2*('|' + 4*(' ')) + '|'))
+    print((2*('|' + 4*(' ')) + '|'))
+    print((2*('|' + 4*(' ')) + '|'))
+    print((2*('|' + 4*(' ')) + '|'))
+    print(2*('+' + 4*('-')) + '+')
+    print((2*('|' + 4*(' ')) + '|'))
+    print((2*('|' + 4*(' ')) + '|'))
+    print((2*('|' + 4*(' ')) + '|'))
+    print((2*('|' + 4*(' ')) + '|'))
+    print(2*('+' + 4*('-')) + '+')
+
+
+def print_grid(n):
+    '''
+    This function will print a grid with four quadrants of an arbritrary size.
+    The argument will define the numbers of rows/columns in the grid.
+    If the argument needs to be adjusted to provide equal sized quadrants, it
+    will be increased by a value of 1.
+    A '+' character will be used in the corners
+    A '-' or '|' character will be used in between the corners
+    '''
+    # Adjust the argument if necessary to ensure all four quadrants will have
+    # equal sizes.
+    if n % 2 == 1:
+        pass
+    else:
+        n = n + 1
```

Comment:
Also you can just put a "not" in front so you don't have to use a "pass" and then do n += 1

Reply:
Just tested that out and see what you mean.  Using 'not' is much more efficient.  Thanks!